frontend/src/app/modules/interventions/details/details.component.html

```diff
@@ -40,7 +40,7 @@ <h2 class="mat-h2">Załączniki</h2>
       <!-- TODO:  Temporary solution - UX-friendly attachment list with download option needed in the near future -->
       <p *ngIf="!intervention.attachments || !intervention.attachments.length">Brak załączników</p>
       <p *ngFor="let file of intervention.attachments">
-        <a href="{{ createFileUrl(file.id) }}" rel="noopener noreferer" download>
+        <a href="#" (click)="fileDownload.emit(file); $event.preventDefault()">
```

Comment:
I wouldn't create it as a `<a>` tag which looks like a like. People are use to clicking cmd or right clicking and selecting "open in a new tab" 
Maybe plain name and download `<button>` next to it. 
![image](https://user-images.githubusercontent.com/119451/81849316-a8cdb100-9556-11ea-957c-95886e8e8aca.png)

Reply:
I agree Kuba. This is temporary and will be replaced in my next PR. Main point of this PR is the logic and mechanics in service to download a file - not a UI representation